ui/src/components/admin/users.vue

```diff
@@ -143,58 +145,57 @@ export default {
                     "admin" : "Allows user to perform administrative tasks on authorization service"
                 }
             },
-
-            //query: "",
-
             mode: "ui",
             form: null,
-            json : null,
-
+            json: null,
             config: Vue.config,
         }
     },
-
     mounted() {
-        //TODO - only load users in the page
-        this.$http.get(Vue.config.auth_api+"/users").then(res=>{
-            this.users = res.data;
-        }).catch(err=>{
-            console.error(err.response);
-            this.$notify({type: "error", text: err});
-        });
+        this.load();
     },
-
-    methods: {
-        closeModal() {
-            this.$root.$emit('bv::hide::modal', 'modal-useredit')
+    watch: {
+        query() {
+            this.load();
+        },
+        currentPage() {
+            this.load();
+        }
+    },
+    methods : {
+        load() {
+            let skip = 0;
+            const limit = 50;
+            if(this.currentPage > 1) skip = 50 * (this.currentPage - 1);
+            this.$http.get(Vue.config.auth_api+'/users', {params:{
+                find: JSON.stringify({
+                    $or: [
+                        //need to use iLike with postgres..
+                        {fullname: {$regex: this.query, $options : 'i'}},
+                        {email: {$regex: this.query, $options : 'i'}},
+                        {username: {$regex: this.query, $options : 'i'}},
+                        {"profile.public.position": {$regex: this.query, $options : 'i'}},
```

Comment:
I don't think we need to search into profiles. You can just search fullname / email / username